buildroot/support/download/git

```diff
@@ -1,3 +1,8 @@
+#/home/noobs/buildroot/support/download/dl-wrapper -b git -o ./linux-f70eae405b5d75f7c41ea300b9f790656f99a203.tar.gz -H linux//linux.hash -- git://github.com/raspberrypi/linux.git f70eae405b5d75f7c41ea300b9f790656f99a203 linux-f70eae405b5d75f7c41ea300b9f790656f99a203
+#git clone -v --bare git://github.com/raspberrypi/linux.git  linux-f70eae405b5d75f7c41ea300b9f790656f99a203
+#在41行 clone linux内核时 不仅把某一提交版本拉下来并且还把整个仓库都拉了下来，速度太慢
+#在脚本中尝试了浅层克隆，但是由于某种原因失败了，改成了全仓库克隆，克隆了将近3g的仓库
```

Comment:
Ermmm, what?

Reply:
well,this command
"${GIT} ls-remote "${repo}" "${cset}" 2>&1"
even if this branch exist,it whill never be find.

 this command is effective
 branch="`${GIT} ls-remote "${repo}" | grep -i "${cset}" | awk -F '/' '{print $3}'`"

Reply:
Actually I made all the references for myself, you can ignore references,thx

Reply:
I am looking forward the further information

Reply:
Sorry, I was confused by the chinese characters, and didn't realise they were commented-out.